src/AppBundle/Command/ListUsersCommand.php

```diff
@@ -34,12 +34,20 @@
  *
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  */
-class ListUsersCommand extends ContainerAwareCommand
+class ListUsersCommand extends Command
 {
-    /**
-     * @var ObjectManager
-     */
     private $entityManager;
+    private $mailer;
+    private $emailSender;
```

Comment:
You are right ... but at the same time, the type-hint is useful for objects or arrays of objects. In this case, even if the IDE knows this is a string ... there's nothing to complete or help us, right?